mono/metadata/threadpool-ms.c

```diff
@@ -749,7 +749,9 @@ worker_try_create (void)
 	current_ticks = mono_100ns_ticks ();
 	now = current_ticks / (10 * 1000 * 1000);
 	if (0 == current_ticks) {
+#ifndef IL2CPP_ON_MONO
```

Comment:
This warning randomly shows up in the output, and that causes our tests to fail.